approach.md

```diff
@@ -3,8 +3,28 @@ layout: page
 title: Our Approach
 ---
 
-_Distributed Press_ is a proposal to build an open-source publishing tool for the distributed web. Our approach is to engage communities actively researching the many issues we face in content publishing today to create a tool that prioritizes content authors. In particular, we hope to learn from independent media that are practicing alternative publishing models, to investigate if and how the distributed web may augment this ecosystem.
+_Distributed Press_ is an initiative to build an open-source publishing tool for the distributed web.
 
-Beginning this year, we interviewed activists and journalists who combat misinformation and censorship, founders of progressive platforms like Popula and Are.na, builders of open protocols from the IndieWeb and DWeb movement. Some of these interviews are published on [GitHub](https://github.com/benhylau/distributed.press/blob/master/interviews.md). As our project develops, we will continue to collaborate with these communities and many other neighbors.
+Our approach begins with engaging communities actively researching the wide-ranging issues we face in content publishing today, learning from independent media that are practicing alternative models of publishing and governance, as well as progressive technology standards and experimental platforms.
+Our project team of creators and technologists collaborate in a tight feedback loop, constantly integrating our respective perspectives into the initiative to ensure the tool we build will serve creators.
 
-If you are interested in having a conversation about possible ways we may collaborate, [please get in touch](mailto:hello@distributed.press)!
\ No newline at end of file
+## Phase 1: Learning about the ecosystem
+
+Beginning this year, we interviewed founders of progressive platforms like [Popula](https://popula.com) and [Are.na](http://are.na), activists and journalists who combat misinformation and censorship, as well as proponents and builders of open protocols from the IndieWeb and DWeb movement.
+The discussions covered a lot of areas relating to content publishing, from tensions between publicity and anonymity, struggles with tools and infrastructures of authorship and distribution, personal experiences of censorship, learnings from experiments of collective governance, value trade-offs of a decentralized future in a world of misinformation, particular technical implementations, and increasingly dominant publishing platforms that undervalue quality while pushing creators toward atomization and precarious labor.
```

Comment:
The discussions covered a range of themes related to publishing: challenges with authoring tools and distribution networks, personal experiences of censorship, experiments with collective governance, growing concerns around misinformation, emerging publishing technologies and technical implementations, and the tensions between finding an audience and preserving anonymity. We heard that many dominant publishing platforms continue to undervalue quality, atomize creators, and increase labor precarity.